src/Data/Message/LocalStorage/localStorageMessageDataClass.ts

```diff
@@ -6,8 +6,15 @@ async function attributes(): Promise<DataClassSchema> {
   const lang = await load(currentLanguage)
 
   return {
-    text: ['string', { title: 'Text' }],
     createdAt: ['string', { title: lang === 'de' ? 'Gesendet am' : 'Sent at' }],
+    createdBy: [
+      'reference',
+      {
+        title: lang === 'de' ? 'Gesendet von' : 'Send by',
```

Comment:
* [ ] It must be "Sent by". If we got these localizers from the backend, they must be fixed there, too.

Reply:
This was only a bug in the portal app, not in the backend.